### ISSUE TITLE
Incorrect removing card from other fragment

### ISSUE THREAD

**Issue by stasusov** [dexafree/MaterialList#76]
I have ViewPager with some fragments and identical cards in them. When i dismiss card from first, it is also removing from second fragment.

**Comment by stasusov** [comment]
I'm using simple class extending FragmentPagerAdapter with them

**Comment by dexafree** [comment]
Identical cards means same objects?

If you remove an object from the list, and you use the same source for different lists, then all the lists will reflect that change.

I you do not want that to happen, you should create the cards for each list, instead of sharing the same objects (But that's OOP related, not directly with MaterialList)

**Comment by stasusov** [comment]
I wasn't correct. I have some dynamically created tabs. In each i'm using same simple method to create and add cards by clicking on the button. Then, when I dismiss card in one tab, it is also removing from the second. So, as i suppose, cards in this example are not the same objects.
Identical cards meant they are looking same, have same information in it.

**Comment by dexafree** [comment]
Doy you create new cards in every tab?
(Calling new Card/BasicButtonsCard.... creating NEW objects, not getting them from an array)

That might be the problem :worried:

**Comment by stasusov** [comment]
No, every time i'm using `Somecard sc = new SomeCard()` where `Somecard` is inherited from `SimpleCard` by all rules

**Comment by dexafree** [comment]
Ok, this is a specific case, I think.

Are you importing the library by module or by gradle.

I'd like to see the output of these line in every MaterialList instance: https://github.com/dexafree/MaterialList/blob/master/materialList/src/main/java/com/dexafree/materialList/view/MaterialListView.java#L169

(sincerely, I have not tested having more than one instance of MaterialList simultaneously running)

**Comment by stasusov** [comment]
If I haven't missed something, this method wasn't even called
